autonity/solidity/test/protocol.js

```diff
@@ -14,6 +14,18 @@ const ValidatorState = {
   jailed : 2
 }
 
+const midSeveritySlashingRatePercent = 10;
+
+async function modifiedSlashingFeeAccountability(autonity, accountabilityConfig, operator, deployer) {
```

Comment:
```suggestion
async function customSlashingRate(autonity, accountabilityConfig, operator, deployer) {
```
No need to specify it is for accountability, slashing is part of accountability.

Reply:
I just saw it returns accountability, it makes sense then.